dias/analyzers/feedpositions_analyzer.py

```diff
@@ -392,7 +414,15 @@ def __east_west_positions(self, src):
                         time, evec[f, i, :], freq[f], np.radians(dec),
                         pad_fac=self.pad_fac_EW)
 
-        return ew_positions, resolution
+        if self.flag_bad_inputs:
```

Comment:
Should we handle failures here like if flagging broker not running or if it returns sth unexpected and in these cases just not use flags (instead of crashing)?

Reply:
yes, we should ask Seth for his knowledge @ssiegelx

Reply:
At minimum, if the flagging broker isn't running, `requests.post` is going to raise `ConnectionError`.  You should catch that (in a `try..except`) so you can do something appropriate when that happens.